virtualenv.py

```diff
@@ -1472,7 +1475,7 @@ def install_files(home_dir, bin_dir, prompt, files):
         content = content.replace('__VIRTUAL_ENV__', home_dir)
         content = content.replace('__VIRTUAL_NAME__', vname)
         content = content.replace('__BIN_NAME__', os.path.basename(bin_dir))
-        writefile(os.path.join(bin_dir, name), content)
+        writefile(os.path.join(bin_dir, name), content, use_system_locale=(name.endswith('.bat')))
```

Comment:
I'm sorry, but ```getfilesystemencoding``` simply won't work for Windows BAT files. I understand your point that ```sys.stdout.encoding``` may not be available, and I don't actually know a way of getting the "OEM" code page if that's the case. But that's what you need to do - using ```getfilesystemencoding``` is just as broken (on for example a UK English system) as using UTF-8.